src/patternfly/components/Card/card.scss

```diff
@@ -296,19 +299,19 @@
     --pf-c-card__header-toggle--MarginLeft: var(--pf-c-card__header--m-toggle-right--toggle--MarginLeft);
 
     .pf-c-card__actions {
-      --pf-c-card__header-toggle--MarginRight: var(--pf-c-card__header--m-toggle-right--actions--MarginRight);
+      --pf-c-card__actions--MarginRight: var(--pf-c-card__header--m-toggle-right--actions--MarginRight);
```

Comment:
Looks like this variable doesn't exist, but the result is expected as `.pf-c-card__header-toggle` has a left margin update for `.pf-c-card__header.pf-m-toggle-right`, therefore this looks like it's unnecessary.

Reply:
Which variable are you saying doesn't exist?